components/Layout/Footer.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC } from "react";
+import Image from "next/image";
+import {
+  Flex,
+  Box,
+  Link,
+  VStack,
+  HStack,
+  Button,
+  NAMED_COLORS,
+  Text,
+  useColorModeValue
+} from "@ironfish/ui-kit";
+import {
+  TelegramIcon,
+  GitHubIcon,
+  RedditIcon,
+  DiscordIcon,
+  TwitterIcon,
+  IronFishLogo,
+} from "svgx";
+
+const Footer: FC = () => {
+  const colors = useColorModeValue(
+    {
+      bg: NAMED_COLORS.WHITE,
+      border: "#E1E1E1",
+      color: NAMED_COLORS.DEEP_BLUE
+    },
+    {
+      bg: NAMED_COLORS.LIGHT_BLACK,
+      border: NAMED_COLORS.DARK_GREY,
+      color: NAMED_COLORS.PALE_GREY
+    },
+  )
+
+  return (
+    <Box
+      w="100%"
+      h="20rem"
+      bgColor={colors.bg}
+      borderTop="0.0625rem solid"
+      borderColor={colors.border}
+      p="4rem 7.6875rem 0rem 7.6875rem"
```

Comment:
Minor but I think you can skip the final `left` padding value and it will be inferred.